src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -211,7 +211,7 @@ public function getPath()
      */
     public function setPath($path)
     {
-        $this->path = $path;
+        $this->path = $path ?? '';
```

Comment:
This looks wrong. The phpdoc says you must pass a string. Any code that calls this with null is wrong and needs to be fixed.

Reply:
Should this test just be removed, then?

Reply:
I think it's not a good idea to send it as a string when it is empty.  And I don't find it right to make these changes before this 8.1 is released. All together, we should see what happens when that version arrives and act that way, we may be making a lot of changes unnecessarily now.

Reply:
@selcukcukur 

> And I don't find it right to make these changes before this 8.1 is released.

Thankfully the maintainers do.

> we may be making a lot of changes unnecessarily now

I've provided you with links to accepted RFC in reply to your comments on my previous PR.